main.py

```diff
@@ -355,8 +355,9 @@ def get_cache_client():
     # It's alright, pylint: disable=global-statement
     global _CACHE_CLIENT
     if _CACHE_CLIENT is None:
+        bucket_name = os.environ.get('CACHE_BUCKET_NAME')
```

Comment:
Could you follow the pattern of storing the bucket name in an (appropriately-named) global variable, same way as the rest of the clients (database, Pub/Sub, etc.) get their parameters?

Reply:
okay